solidity/hardhat.config.ts

```diff
@@ -31,6 +31,8 @@ const config: HardhatUserConfig = {
 
   networks: {
     hardhat: {
+      allowUnlimitedContractSize: true,
```

Comment:
Let's add a TODO we do it only temporarily for `BridgeStub` and that line needs to be removed in future.

Also, is this line needed given that we `except: ["BridgeStub$"]` in the contract sizer plugin config?

Reply:
Done in b0eb1f3c825cee422c76c8b0d5fdccdf88397f7a.